### ISSUE TITLE
Please make all of our RTD projects public

### ISSUE THREAD

**Issue by mechanomi** [readthedocs/readthedocs.org#7484]
hello! we (https://github.com/crate) love your new GitHub PR build integration! thanks so much! such a wonderful and useful feature

however, we have run into a problem. most of our repositories have some sort of privacy flag on them meaning that the builds are not visible unless you're a maintainer of the project. that's not ideal for us, because these are open source projects and we want to encourage contributions from non-employees

to that end, we'd like our build logs visible to everyone, as is already the case with some of our repos

I've tried to find a way to make all of our repositories "public" but it looks like you removed that option from the admin interface

here's a list of all our projects:

- [ ] https://readthedocs.org/projects/crash/
- [ ] https://readthedocs.org/projects/crate/
- [ ] https://readthedocs.org/projects/crate-admin-ui/
- [x] https://readthedocs.org/projects/crate-clients-tools/
- [ ] https://readthedocs.org/projects/crate-cloud-howtos/
- [ ] https://readthedocs.org/projects/crate-cloud-reference/
- [x] https://readthedocs.org/projects/crate-cloud-tutorials/
- [ ] https://readthedocs.org/projects/crate-dbal/
- [x] https://readthedocs.org/projects/crate-docs-build/
- [ ] https://readthedocs.org/projects/crate-docs-theme/
- [ ] https://readthedocs.org/projects/crate-howtos/
- [ ] https://readthedocs.org/projects/crate-jdbc/
- [ ] https://readthedocs.org/projects/crate-npgsql/
- [ ] https://readthedocs.org/projects/crate-pdo/
- [ ] https://readthedocs.org/projects/crate-python/
- [x] https://readthedocs.org/projects/crate-tutorials/
- [ ] https://readthedocs.org/projects/croud/
- [ ] https://readthedocs.org/projects/sql-99/

I've marked the projects "done" if they already have publicly accessible builds for whatever reason

can someone help me by making the rest of the projects listed public, please?

**Comment by mechanomi** [comment]
cc @mfussenegger

**Comment by humitos** [comment]
I just checked the `crate` project and it seems it's marked as `Protected`, that's why it not possible to access the build output pages. These projects need to be `Public` for that. We removed the privacy levels from our community site in https://github.com/readthedocs/readthedocs.org/pull/7013. However, we didn't update the privacy of these projects.

If @mfussenegger (or anyone listed at https://github.com/orgs/crate/people) confirms that you want all of those projects to be public, I can make that change. (@mechanomi I'm sorry, I didn't find you there :sweat_smile:  )

> love your new GitHub PR build integration! thanks so much!

:heart_eyes:
